packages/rspeedy/core/src/plugins/index.ts

```diff
@@ -64,7 +64,7 @@ export async function applyDefaultPlugins(
     ),
 
     import('./sourcemap.plugin.js').then(({ pluginSourcemap }) =>
-      pluginSourcemap()
+      pluginSourcemap(config.output)
```

Comment:
Would you please add some tests in `packages/rspeedy/core/test/plugins/sourcemap.plugin.test.ts`?